src/Umbraco.Core/Models/ContentEditing/UserDisplay.cs

```diff
@@ -17,12 +17,12 @@ public UserDisplay()
             AvailableCultures = new Dictionary<string, string>();
             StartContentIds = new List<EntityBasic>();
             StartMediaIds = new List<EntityBasic>();
-            Navigation = new List<EditorNavigation>();
+            Navigation = new List<ContentApp>();
```

Comment:
The change to ContentApp makes sense (and is consistent with ContentEditorDisplay), but I'm wondering now if the Navigation property shouldn't also be renamed to ContentApps, also consistent with ContentEditorDisplay and UserGroupDisplay...

Reply:
But would this then be considered a breaking change? Don't know if users are able to get ahold of this UserDisplay

Reply:
Yep, as it is a public property on a public class, this would be considered a breaking change 🛑

Reply:
So, just renaming the Navigation property to ContentApps? Or creating a new property ContentApps and somehow make Navigation obsolete?

Reply:
Yea renaming would be a breaking change, if someone has reffered to this in their code, a rename would break their project :) I would as you said create a new property and obsolete the old 👍

Reply:
Confused me a bit looking across the different implementations - how does Navigation differ from ContentApps? I think they're used interchangeably on different dashboards/views...

Aren't they the same thing (as in, different properties, but aren't content apps the navigation)?

If it's a breaking change regardless, shouldn't it be as clean and tidy as possible and part of 11?

Reply:
It is a breaking change regardless, because we want to change the type from `List<EditorNavigation>` to `List<ContentApp>`, we can't do this as this might break someones project if they are reffering to Navigation in their code.
And no, sadly this would be for V12, as we have to obsolete things with 2 majors notice.

As for how it differs I'm not so sure, i haven't looked closely into this PR or area in general, i was just commenting on whether this would be a breaking change or not because I'm nosy 🕵️

Reply:
Yeah, 12. I can't count!

Reply:
Updated.
I've reverted the changes for the Navigation property and made them obsolete. 
And moved all the content app logic to .ContentApps like the other content apps have as well